tests/vmi_networking_test.go

```diff
@@ -217,14 +218,14 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 			defer expecter.Close()
 
-			addrShow = "ip address show eth0\n"
+			addrShow = "ip address show eth0\r"
 			resp, err := expecter.ExpectBatch([]expect.Batcher{
```

Comment:
generally speaking, just taking this as an example
when there is a repeating prompt string i would suggest please to assign a var and use that var if possible

Reply:
When you say "prompt string" do you mean the `\r`? Are you suggesting to keep the `\r` a constant in the utils file?

Reply:
i mean "\\$ " , not the \r
console prompt